lib/Yield/PTML.cpp

```diff
@@ -535,19 +543,71 @@ static std::string taggedText(const PTMLBuilder &ThePTMLBuilder,
   return Result;
 }
 
+static std::string instructionPrefix(const PTMLBuilder &ThePTMLBuilder,
+                                     const yield::Instruction &Instruction) {
+  std::string Result;
+
+  if (options::InstructionAddressColumnWidth) {
+    std::string SerializedAddress = Instruction.Address().toString();
+    Result += ThePTMLBuilder.getTag(tags::Span, SerializedAddress)
+                .addAttribute(attributes::Token, tokenTypes::InstructionAddress)
+                .serialize()
+              + ' ';
+
+    if (SerializedAddress.size() < options::InstructionAddressColumnWidth) {
+      uint64_t Column = options::InstructionAddressColumnWidth;
+      Result += ThePTMLBuilder
```

Comment:
I'd be really happy if I could have a commit that `s/ThePTMLBuilder/B/g` in the whole codebase.

Reply:
Let's move this to the model PR instead.